docs/scripts/check-closing-quotes.sh.whitelist

```diff
@@ -0,0 +1,10 @@
+# Every line of this file is a regex. Each regex is ignored when checking unpaired quotes.
+# You can add comments to this file if the comment line is unlikely to match.
+# Just do not add too generic patterns (such as empty line) because they would filter out too many lines.
+^      _\\ \\/ _ `/ _ \\/ _  / _ \\/ _ \\\\ \\ /$
+^   _\\ \\/ _ `/ _ \\/ _  / _ \\/ _ \\\\ \\ /$
+^      /    / _ `/ |/ / / _ `/ _ `/ __/ _ \\/ __/$
+^     /    / _ `/ |/ / / _ `/ _ `/ __/ _ \\/ __/$
+^```````````````$
+^```````````````````$
+^      ```$
```

Comment:
This shouldn't be whitelisted, it's not valid markup. The Bifunctor module needs changing as it doesn't render properly in our docs.

Reply:
Removed and attempted to fix Bifunctor. Appears to be okay locally.

Reply:
See my suggestion below. Since code blocks in daml-doc generated docs need triple backticks, we need to whitelist it after all.